doc/getting_started.md

```diff
@@ -581,6 +581,56 @@ browser.
 
 ![](img/sample_traces_kmac.png)
 
+### SHA3 Capture
+
+To perform a SHA3 capture, use this command:
+```console
+$ cd cw
+$ ./capture.py --cfg-file capture_sha3_cw310.yaml capture sha3-random --num-traces 100 --plot-traces 10
+```
+
+The above command will send SHA3 requests with random texts.
+In order to capture traces for Fixed-vs.-Random Datasets, the following command may be used:
+```console
+$ cd cw
+$ ./capture.py --cfg-file capture_sha3_cw310.yaml capture sha3-fvsr-data --num-traces 100 --plot-traces 10
+```
+
+You should see similar output as in the KMAC example. Once the power traces have
+been collected, a picture similar to the following should be shown in your
+browser.
+
+![](img/sample_traces_sha3.png)
+
+
+You can also use the batch mode with
+```console
+$ cd cw
+$ ./capture.py --cfg-file capture_sha3_cw310.yaml capture sha3-fvsr-data-batch --num-traces 100 --plot-traces 10
+```
+
+However, there might occur errors due to synchronization issues. In this case try to reset the board and
+try again.
+
+To disable masking use the ```capture_sha3_masks_off_cw310.yaml``` file. The capture commands stay the same.
+
+
+
+### SHA3 TVLA
+
+To perform the tvla just run
+```console
+$ cd cw
+$ ./tvla.py --cfg-file tvla_cfg_sha3.yaml run-tvla
+```
```

Comment:
This will run the general (non-specific) TVLA, which is the only one that we support right now.
It might be useful to mention that this analysis should be used only on Fixed-vs.-Random Datasets.

Reply:
Good point. Done.